bower.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "ilios-calendar",
   "dependencies": {
-    "ember": "~2.8.0",
+    "ember": "~2.9.0-beta.1",
```

Comment:
alright. let's sit on it for a moment, then. until new, stable versions of ember/ember-ds become available.